.travis.yml

```diff
@@ -16,7 +19,18 @@ sudo: required
 services:
   - docker
 
+before_install:
+  # install onigmo
+  - cd ${HOME}
+  - wget https://github.com/k-takata/Onigmo/releases/download/Onigmo-6.2.0/onigmo-6.2.0.tar.gz -O onigmo-6.2.0.tar.gz
+  - tar zxvf onigmo-6.2.0.tar.gz
+  - cd onigmo-6.2.0
+  - ./configure
+  - make
+  - sudo make install
```

Comment:
We should also update the docs about how to install onigmo. Isn't there a more user-friendly way of installing it?

Reply:
I've updated a README, as well.